homeassistant/components/system_bridge/model.py

```diff
@@ -0,0 +1,22 @@
+"""Type definitions for System Bridge integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Callable
+
+from homeassistant.components.binary_sensor import BinarySensorEntityDescription
+from homeassistant.components.sensor import SensorEntityDescription
+
+
+@dataclass
+class SystemBridgeBinarySensorEntityDescription(BinarySensorEntityDescription):
```

Comment:
Please define the descriptions in the corresponding platforms where they are used. These are platform details.